efficiency/bin/sdl_timing

```diff
@@ -114,6 +114,8 @@ if [ -z ${SPECIFICGPUVERSION} ] || [[ "${SPECIFICGPUVERSION}" == "explicit_cache
 fi
 
 echo "Total Timing Summary"
-echo "   Evt    Hits       MD       LS      T3       T5       pLS       pT5      pT3      TC       Event      Short           Rate"
-#echo "   Evt    Hits       MD       LS      T3       T5       pLS       pT5      pT3      TC       TCE      Event      Short           Loop      Effective"
+grep -h "Time for map " timing_temp.txt | cut -d " " -f 6 | awk '{ SUM += $1} END { print "Average time for map loading =",SUM/5,"ms" }' # 5 is the number of stream values run
```

Comment:
What do you mean by "5 is the number of stream values run"?

Reply:
The time is sampled 5 times:
1. Run w/ 1 stream
2. Run w/ 2 stream
3. Run w/ 4 stream
4. Run w/ 6 stream
5. Run w/ 8 stream

So it's total time of 5 runs divided by the number of runs (=5).

Reply:
Thank you! Makes sense.